### PR TITLE
MG-198 - Add post-deployment guide for k8s

### LINKED ISSUE #198: Feature: MG-198 - Add post-deployment guide for k8s
### Is your feature request related to a problem? Please describe.

Documentation on how to interact with the DigitalOcean Kubernetes (DO k8s) cluster from a local deployment of Magistrala

### Describe the feature you are requesting, as well as the possible use case(s) for it.

Detailed documentation that outlines the steps required to configure and interact with a DO k8s cluster from a local deployment of Magistrala. 

### Indicate the importance of this feature to you.

Must-have

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by JeffMboya**
### What does this do?
This PR adds a section to the README file explaining how to interact with Magistrala services after K8s deployment, both through the web-based User Interface (UI) and via Postman. 

### Which issue(s) does this PR fix/relate to?
- Resolve #198

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### Notes
This update is important for users deploying Magistrala on a Kubernetes cluster, particularly with a DigitalOcean load balancer, as it provides clear steps for accessing and interacting with the deployed services.

### CODE REVIEW

File: docs/kubernetes.md [hunk 1]
```diff
@@ -202,7 +202,66 @@ kubectl get services -n mg
 kubectl logs <pod-name> -n mg
 ```
 
----
+### Interacting with Magistrala Services After Deployment
+
+Once you have successfully deployed Magistrala, there are two primary ways you can interact with its services:
+
+- through the web-based User Interface (UI)
+- via Postman
```
Comment (dborovcanin) [inline docs/kubernetes.md]: Once you have successfully deployed Magistrala, there are three primary ways you can interact with its services:

- web-based User Interface (UI)
- Magistrala CLI tool
- HTTP API Clients (e.g., cURL, Postman)

File: docs/kubernetes.md [hunk 2]
```diff
@@ -202,7 +202,66 @@ kubectl get services -n mg
...
+- Once you have the Public IP address, open your web browser.
+- In the address bar, enter the IP address followed by `/ui/login` as shown below:
+
+      http://138.68.126.8/ui/login
+
+#### Using Postman
+
+If you prefer working with APIs, you can also interact with Magistrala services using Postman by sending requests to the Public IP address of your load balancer. For example, to create a user:
+
+###### 1. Set Up the Postman Request
```
Comment (dborovcanin) [inline docs/kubernetes.md:236]: Upload a Postman collection (you can even generate it from the API docs) and add a link to it.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1 +1 @@
-*   @absmach/maintainers
+- @absmach/maintainers
```
Comment (arvindh123) [inline .github/CODEOWNERS]: Revert this back to original

**Review by arvindh123 — CHANGES_REQUESTED** [review]

File: docs/kubernetes.md [hunk 4]
```diff
@@ -202,7 +202,65 @@ kubectl get services -n mg
 kubectl logs <pod-name> -n mg
 ```
 
----
+### Interacting with Magistrala Services After Deployment
+
+Once you have successfully deployed Magistrala, there are three primary ways you can interact with its services:
+
+- web-based User Interface (UI)
+- Magistrala CLI tool
```
Comment (dborovcanin) [inline docs/kubernetes.md]: Add a link to the CLI docs at https://docs.magistrala.abstractmachines.fr/cli.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**